README.md

```diff
@@ -1,3 +1,8 @@
 # TextMining
 
 This is the base repo for the text mining and analysis project for Software Design at Olin College.
+
+# commonwords.py
+Required packages:
+
+$ pip install wikipedia
```

Comment:
I would put more documentation on README file.
Besides required packages, you can put descriptions, relevant links, or how to use it